nav2_dwb_controller/dwb_critics/src/map_grid.cpp

```diff
@@ -47,17 +47,8 @@ namespace dwb_critics
 {
 
 // Customization of the CostmapQueue validCellToQueue method
-bool MapGridCritic::MapGridQueue::validCellToQueue(const costmap_queue::CellData & cell)
+bool MapGridCritic::MapGridQueue::validCellToQueue(const costmap_queue::CellData & /*cell*/)
 {
-  unsigned char cost = costmap_.getCost(cell.x_, cell.y_);
-  if (cost == nav2_costmap_2d::LETHAL_OBSTACLE ||
-    cost == nav2_costmap_2d::INSCRIBED_INFLATED_OBSTACLE ||
-    cost == nav2_costmap_2d::NO_INFORMATION)
-  {
-    parent_.setAsObstacle(cell.index_);
```

Comment:
If it's a short term fix, instead of deleting all the code, I'd rather comment these two lines, add a TODO comment on top and probably log a warning.

Reply:
Generally I would be for removing non-active code. However, as Carlos mentions, if it's a short-term work-around and the code will soon be restored, I'd leave it for documentation purposes to help the reader understand the context.

Reply:
For the long term fix, I think this will be gone forever in the goal critics. For the path critics, I'm not sure if this would come back. I believe the bug is still there for those critics. It just doesn't turn in to an obvious defect because of the redundancy of having multiple points in a path.